home/views/admin.py

```diff
@@ -21,10 +21,8 @@
 from home.utils import send_email, _ttl_cache
 from planetterp import config
 
-class Admin(UserPassesTestMixin, View):
-
-    def test_func(self):
-        return self.request.user.is_staff
+class Admin(PermissionRequiredMixin, View):
+    permission_required = "home.admin"
```

Comment:
Thoughts on making an enum to store the string value? I assume the string won't change, but I think it's more pleasant to deal with than a string. Also, this would give us more flexibility in naming/updating it.

Reply:
this doesn't work as well as you might hope because there's no way to use this enum value in templates. Currently django populates a global `perms` variable and does some magic for perms checking:

```
{% if perms.home.admin %}
...
{% endif %}
```

I don't even know how this would work with a custom enum. In addition, we would have to populate the template with our `Permission` enum, which requires a custom context processor. 

I'm not sure the added complexity is worthwhile here.